rosbag2_transport/include/rosbag2_transport/play_options.hpp

```diff
@@ -24,8 +24,11 @@ namespace rosbag2_transport
 struct PlayOptions
 {
 public:
+  using TopicFilter = std::function<bool (const std::string & topic)>;
+
   size_t read_ahead_queue_size;
   std::string node_prefix = "";
+  TopicFilter exclude_topic_filter = [](const std::string &) {return false;};
```

Comment:
When I am thinking about this, I believe it's a bit overkill to provide a lambda/function for this. Why can't a simple `std::vector<std::string>` not suffice? Especially when using this as an user facing option. I believe a list of vectors is easier and we can internally apply a generic lambda.